roles/servicetelemetry/templates/manifest_grafana_v5.j2

```diff
@@ -0,0 +1,95 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: {{ ansible_operator_meta.name }}-grafana
```

Comment:
The route name comes from the name of the Grafana instance due to how the manifest is configured now. I couldn't find a great way to override this without having to add more RBAC capabilities so the Operator could read and parse the cluster configuration to get the external hostname for routes so I could construct the entire `host` parameter. I didn't feel it was worth that complexity, so we'll have a slightly different route name in our documentation now.

Reply:
Yes, the dashboard URL will change. Thanks for the reminder. I did think of this for the release notes, but I haven't written it down yet. Now that I have the PR up, I'm going to stage a release note in the main epic that covers this.